profiler/profiler.heapwalker/src/org/netbeans/modules/profiler/heapwalk/ClassesListController.java

```diff
@@ -164,8 +164,7 @@ public Object[][] getData(String[] filterStrings, int filterType, boolean showZe
                     else data[i][4] = (retainedSizeByClass > 0 ? "+" : "") + numberFormat.format(retainedSizeByClass); // NOI18N 
                 }
             } else {
-                data[i][1] = new Double((double) instancesCount /
-                                     (double) totalLiveInstances * 100);
+                data[i][1] = (double)instancesCount / totalLiveInstances * 100;
```

Comment:
you can drop the double if you want. right hand side is double, which means the result is a double